testsuite/MDAnalysisTests/topology/test_itp.py

```diff
@@ -283,25 +365,31 @@ def test_defines(self, top):
         assert_allclose(top.charges.values[1], 0.241)
         assert_allclose(top.charges.values[2], 0.241)
 
-    
+
 class TestITPKeywords(TestITPNoKeywords):
     """
     Test reading ITP files *with* defined keywords.
     """
+    expected_attrs = ['ids', 'names', 'types',
+                      'charges', 'chargegroups',
+                      'resids', 'resnames',
+                      'segids', 'moltypes', 'molnums',
+                      'bonds', 'angles', 'dihedrals', 'impropers', 'masses', ]
+    guessed_attrs = ['elements', ]
```

Comment:
This should just be inherited from TestITPNoKeywords, unless I'm missing something -- best not to repeat code where possible.